Automattic/wp-calypso
# Remove `marketplace-reviews-show` feature flag

Closes https://github.com/Automattic/dotcom-forge/issues/5190

## Proposed Changes

* Remove the `marketplace-reviews-show` feature flag and all its usages

## Testing Instructions



* Go to a plugin page you can review with the feature flag disabled, eg. `/plugin/woocommerce-bookings?flags=-marketplace-reviews-show`
* Check that the review banner and review cards are shown on the page
  ![CleanShot 2024-01-23 at 11 18 07@2x](https://github.com/Automattic/wp-calypso/assets/11555574/d3797584-73f0-4a59-a5de-a92ef66c3845)
  ![CleanShot 2024-01-23 at 11 18 13@2x](https://github.com/Automattic/wp-calypso/assets/11555574/d832e9a8-3356-4e05-91d0-e37fabaff889)


## Pre-merge Checklist
